multidim_threshold/threshold.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+from copy as deepcopy
+
+from .refine import box_edges
+from .rectangles import to_rec
+
+# Return False only if the polarity is wrong
+compute_var_polarity = lambda stleval, bot, top: stleval(bot) <= stleval(top)
```

Comment:
perhaps name: `has_positive_polarity`.
This makes it obvious what `True` and `False` should signify.

Also, I'd recommend defining using def instead of a lambda. Linters prefer it.